.github/workflows/test.yml

```diff
@@ -43,7 +43,7 @@ jobs:
     - name: Test
       run: go test ./...
     - name: Test play dist
-      run: NETLIFY=true NETLIFY_BUILD_BASE=$(mktemp -d) CUELANG_ORG_DIST=$(mktemp -d) bash dist.sh
+      run: NETLIFY=true NETLIFY_BUILD_BASE=$(mktemp -d) bash build.sh
```

Comment:
Moved all files to be called `*.bash` for consistency, having also adjusted the shebangs. Also dropped the `bash build.sh` as suggested because that is unnecessary as you say.